server/utils/routes.js

```diff
@@ -9,10 +10,18 @@ export const routes = {
   login: `/login`,
   signup: `/signup`,
   logout: `/logout`,
+  createdConfiguration: (id) => `${routes.root}u/configure?s=edit&id=${id}`,
```

Comment:
I don't see where this is used.

Reply:
good find, we don't need this anymore
[d053f38](https://github.com/AMP-SCZ/dpdash/pull/305/commits/d053f38a226f932f599e2b21002daf20a6c9f788)